### PR TITLE
Add team name edit functionality in settings

### PR DESCRIPTION
**Pull request by gentamura**
## Summary
Added interactive team name editing functionality in the settings page. This includes real-time validation, error handling, and server-side updates using Server Actions.

## Related Issue
N/A

## Changes
- Added `actions.ts` for server-side operations
  - Implemented `getTeamName` action for fetching team data
  - Implemented `updateTeamName` action with revalidation
- Added team name editing components
  - Created `team-name.tsx` for interactive editing UI
  - Implemented input validation (1-256 characters)
  - Added real-time error feedback
- Updated settings page layout to include team name editing section

## Testing
- Verified team name updates successfully persist in database
- Confirmed validation works for:
  - Empty input (shows error)
  - Input exceeding 256 characters (shows error)
  - Valid input (updates successfully)
- Tested revalidation functionality after updates

## Other Information
- Uses valibot for input validation
- Implements optimistic updates with error handling
- Uses Server Actions for data mutations
- Maintains consistent UI/UX with the existing components

### CODE REVIEW

**Comment by gentamura** [comment]
@satococoa Code review please 🙏

**Review by toyamarinyon — APPROVED** [review]

File: app/(main)/settings/team/actions.ts [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+	const _teams = await db
+		.select({ dbId: teams.dbId, name: teams.name })
+		.from(teams)
+		.innerJoin(teamMemberships, eq(teams.dbId, teamMemberships.teamDbId))
+		.innerJoin(
+			supabaseUserMappings,
+			eq(teamMemberships.userDbId, supabaseUserMappings.userDbId),
+		)
+		.where(eq(supabaseUserMappings.supabaseUserId, user.id))
+		.execute();
```
Comment (satococoa) [inline app/(main)/settings/team/actions.ts]: You don't need `.execute()`.

In my understanding, `execute()` in Drizzle ORM is necessary in specific cases:

1. To run raw SQL queries directly, e.g., ``db.execute(sql`...`)``.
2. When performing database operations outside the query builder.
3. For advanced operations inside transactions, e.g., custom updates or inserts.
4. To handle SQL syntax unsupported by the query builder.

For more details, refer to the [Drizzle ORM Docs](https://orm.drizzle.team/docs/sql).

**Review by satococoa — COMMENTED** [review]
Thank you for your careful work!
Could you check out my comment, please?

**Comment by gentamura** [comment]
@satococoa I fixed it 👍 [4ecc218](https://github.com/giselles-ai/giselle/pull/127/commits/4ecc21852f74ac4d83d0938d9309902e872160f0)
and updated suspense fallback [4567156](https://github.com/giselles-ai/giselle/pull/127/commits/4567156756f49991f18271e7e4e562658188e8d7)

**Review by satococoa — APPROVED** [review]
I've confirmed the fixes!
Thank you 👍 👍

**Comment by gentamura** [comment]
@shige review, please 🙏

**Review by shige — APPROVED** [review]
LGTM! 📛 

IMO:
I think it would be even more convenient if pressing the Enter key allowed saving!

**Comment by gentamura** [comment]
Thanks for reviewers 🚀

**Pull request merged**